packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionContainerStartCommand.test.ts

```diff
@@ -0,0 +1,102 @@
+import { vscodeStub } from '@salesforce/salesforcedx-utils-vscode/test/unit/commands/mocks';
+import * as proxyquire from 'proxyquire';
+import { assert, createSandbox, SinonSandbox, SinonStub, stub } from 'sinon';
+
+const proxyquireStrict = proxyquire.noCallThru();
+
+describe.only('Force Function Start Container Command Unit Tests.', () => {
+  let sandbox: SinonSandbox;
+
+  before(() => {
+    sandbox = createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe('Verify container start command.', () => {
+    const fakeUri: any = 'what';
+    let validateStartFunctionsUriStub: SinonStub;
+    let commandletRunStub: SinonStub;
+    let forceFunctionContainerStartCommand: any;
+    let FUNCTION_CONTAINER_LOG_NAME: string;
+    let CONTAINER_START_TEXT_KEY: string;
+    let filePathGathererConstructorStub: SinonStub;
+    let sfdxWorkspaceCheckerConstructorStub: SinonStub;
+    let forceFunctionContainerStartExecutorConstructorStub: SinonStub;
+
+    beforeEach(() => {
+      validateStartFunctionsUriStub = stub();
+      const constructorStub = stub();
+      class SfdxCommandlet {
+        constructor(...args: any) {
+          return constructorStub(...args);
+        }
+        public run() {}
+      }
+
+      filePathGathererConstructorStub = stub();
+      class FilePathGatherer {
+        constructor(...args: any) {
+          return filePathGathererConstructorStub(...args);
+        }
+      }
+
+      sfdxWorkspaceCheckerConstructorStub = stub();
+      class SfdxWorkspaceChecker {
+        constructor(...args: any) {
+          return sfdxWorkspaceCheckerConstructorStub(...args);
+        }
+      }
+
+      forceFunctionContainerStartExecutorConstructorStub = stub();
+      class ForceFunctionContainerStartExecutor {
+        constructor(...args: any) {
+          return forceFunctionContainerStartExecutorConstructorStub(...args);
+        }
+      }
+
+      commandletRunStub = sandbox.stub(SfdxCommandlet.prototype, 'run');
+      ({
+        forceFunctionContainerStartCommand,
+        CONTAINER_START_TEXT_KEY,
+        FUNCTION_CONTAINER_LOG_NAME
+      } = proxyquireStrict(
+        '../../../../src/commands/functions/forceFunctionContainerStartCommand',
+        {
+          vscode: vscodeStub,
+          '../util': {
+            FilePathGatherer,
+            SfdxCommandlet,
+            SfdxWorkspaceChecker
+          },
+          './forceFunctionStart': {
+            ForceFunctionContainerStartExecutor,
+            validateStartFunctionsUri: validateStartFunctionsUriStub
+          }
+        }
+      ));
```

Comment:
So here is where I pull in the exported members from the forceFunctionContainerStartCommand module.  This is done here in order to provide the stubbed dependancies to the class under test.